src/config.inc

```diff
@@ -22,7 +22,8 @@ endif
 #CHAFF = ../../zChaff
 #BOOLEFORCE = ../../booleforce-0.4
 #MINISAT = ../../MiniSat-p_v1.14
-MINISAT2 = ../../minisat-2.2.1
+#MINISAT2 = ../../minisat-2.2.1
+#IPASIR = ../../ipasir
```

Comment:
`LIBSOLVER` should be added here, alongside instructions on when to set it, and what to set it to.

Reply:
I am happy to document LIBSOLVER in that file, however, I would not force people to use a solver by default (yet), as all CI will fall apart. I'd make this a followup commit.